demos/projects/ST/b-l475e-iot01a/port/azure_iot_flash_platform.c

```diff
@@ -223,18 +215,18 @@ AzureIoTResult_t AzureIoTPlatform_VerifyImage( AzureADUImage_t * const pxAduImag
 
         for( int i = 0; i < azureiotflashSHA_256_SIZE; ++i )
         {
-            printf( "%x", ucDecodedManifestHash[ i ] );
+            AZLogInfo( ( "%x", ucDecodedManifestHash[ i ] ) );
```

Comment:
Sorry, didn't get a chance to look at this last night, but this will print out the hash on a bunch of lines - not sure if this is what we want
![image](https://user-images.githubusercontent.com/54871851/201169594-6111d6b6-004e-4bf8-b983-0300978f01f6.png)

Reply:
Thanks for bringing this up. It was intentional.
I didn't want to add more memory manipulation just to have the log printed nicely.
The best way would be to change AzLogInfo to have an option to print without adding timestamp and newline, then it would print the way we really would like it to. But I didn't want to make such changes to logging at this point.